db/index.js

```diff
@@ -27,12 +24,12 @@ const tiers = ['Backer', 'Superbacker'];
 const projects = [['Indie Movie', 100], ['Crypto Currency Project', 101], ['Hipster Notebook', 102]];
 
 const randomizer = (arr) => {
-  const random = Math.floor(Math.random * Math.floor(tiers.length))
+  const random = Math.floor(Math.random() * Math.floor(tiers.length))
```

Comment:
I believe you don't need the Math.floor function for the 'tiers.length', an array length is always going to be an entire integer

Reply:
I have recomitted, and there was also a bug where I wasn't checking the input array length but only the hardcoded tiers length. Resolved.